### ISSUE TITLE
[a11y] Cannot activate Share in message options menu via keyboard

### ISSUE THREAD

**Issue by jcsteh** [vector-im/riot-web#12365]
### Description

The menu item to share a message cannot be activated from the keyboard.

### Steps to reproduce
1. Focus/mouse over a message so the actions appear.
2. Choose Options.
3. Press down arrow several times to select Share.
4. Press enter.
    - Expected: The message sharing dialog should appear.
    - Actual: Nothing happens.

### Version information

- **Platform**: web

- **Browser**: Firefox 75 nightly
- **OS**: Windows
- **URL**: chat.mozilla.org

**Comment by jcsteh** [comment]
Interestingly, the other items in the message options menu activate just fine with the enter key.
